source/pkcore/abilitypages/chi/smokescreen.rst

```diff
@@ -0,0 +1,61 @@
+.. smokescreen:
+###########
+SmokeScreen
+###########
+
+Description
+###########
+
+SmokeScreen is a Chi Ability.
+
+SmokeScreen enables a chiblocker to blind their enemies, allowing for a quick escape or an unseen approach. GIF TO BE ADDED
```

Comment:
Change to `SmokeScreen allows Chiblockers to blind all enemies in an area from a distance, giving them a chance for a quick escape or an unseen approach. GIF TO BE ADDED`

Reply:
I think its better to write it as "SmokeScreen enables a chiblocker to blind enemies from a distance, allowing for a quick escape or an unseen approach"